package.json

```diff
@@ -31,8 +31,9 @@
     "node-gyp": "^3.8.0",
     "nyc": "^12.0.2",
     "optimist": "~0.6.1",
-    "prebuildify": "^2.10.0",
+    "prebuildify": "github:ahdinosaur/prebuildify",
     "prebuildify-ci": "^1.0.4",
+    "prebuildify-cross": "^2.0.3",
```

Comment:
Where's the connection between `prebuildify` and `prebuildify-cross`?

Reply:
`prebuildify-cross` will build a docker image for the given target, create a container with the project mounted inside, run `npm install --no-scripts` and then `npm run prebuild`, and then copy out `prebuilds`. so it depends on having a `prebuild` script using `prebuildify`.

Reply:
Thanks for clarifying. I'll have a closer look at `prebuildify-cross` as well. Just had a quick glance and it's some hardcore `docker` magic in there :D

Reply:
@ahdinosaur Shouldn't `--no-scripts` be [`--ignore-scripts`](https://docs.npmjs.com/cli/install)?

Reply:
@vweevers: yes, sorry was just dumping words from memory: https://github.com/ahdinosaur/prebuildify-cross/blob/67f9fe3af4a45769bb2d01d1176face03b51da8f/build-in-docker#L14